lib/settings.h

```diff
@@ -161,7 +161,10 @@ class CPPCHECKLIB WARN_UNUSED Settings {
     std::string clangExecutable = "clang";
 
     /** Use clang-tidy */
-    bool clangTidy{}; // TODO: CLI
+    bool clangTidy{};
+
+    /** Custom clang-tidy executable */
+    std::string clangTidyExecutable = "clang-tidy";
```

Comment:
we could default initialize with the proper value (in windows "clang-tidy.exe") but I don't know if that really matters.

Reply:
I am fine with that reasoning..